core/meta/src/TClass.cxx

```diff
@@ -1075,7 +1075,7 @@ TClass::TClass() :
    fStreamer(nullptr), fIsA(nullptr), fGlobalIsA(nullptr), fIsAMethod(nullptr),
    fMerge(nullptr), fResetAfterMerge(nullptr), fNew(nullptr), fNewArray(nullptr), fDelete(nullptr), fDeleteArray(nullptr),
    fDestructor(nullptr), fDirAutoAdd(nullptr), fStreamerFunc(nullptr), fConvStreamerFunc(nullptr), fSizeof(-1),
-   fCanSplit(-1), fIsSyntheticPair(kFALSE), fProperty(0), fClassProperty(0), fHasRootPcmInfo(kFALSE), fCanLoadClassInfo(kFALSE),
+   fCanSplit(-1), fIsSyntheticPair(kFALSE), fHasCustomStreamerMember(false), fProperty(0), fClassProperty(0), fHasRootPcmInfo(kFALSE), fCanLoadClassInfo(kFALSE),
```

Comment:
This is minor, but shouldn't we use kFALSE to initialise fHasCustomStreamerMember to be consistent with the other booleans initialised in this constructor?

Reply:
We are currently on the fence (Some of the code base has already moved to using `true/false`).  I'll revert back to kFALSE here but I will also likely upload another PR for the conversion.